docs/content/pcloud.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "pCloud"
 description: "Rclone docs for pCloud"
-versionIntroduced: "v1.39"
+versionIntroduced: "v1.40"
```

Comment:
Why this change?

Reply:
aha sry I just thought I may need to update version if it has a new update, let me revert this part back